src/Query/Conditions/BetweenColumnsConditionBuilder.php

```diff
@@ -41,17 +49,23 @@ public function build(ExpressionInterface $expression, array &$params = []): str
      * @param Query|ExpressionInterface|string $columnName
      * @param array $params the binding parameters.
      *
-     * @return string
+     * @throws Exception|InvalidArgumentException|InvalidConfigException|NotSupportedException
+     *
+     * @return array|string
      */
-    protected function escapeColumnName($columnName, array &$params = []): string
+    protected function escapeColumnName($columnName, array &$params = [])
```

Comment:
That should always return a string or else code in `build()` will fail.

Reply:
fixed.